src/main/java/fruitshop/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package fruitshop.service.impl;
+
+import fruitshop.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String file) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file))) {
+            String value = bufferedReader.readLine(); // skip header (first line)
+            while ((value = bufferedReader.readLine()) != null) {
+                data.add(value);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + file, e);
```

Comment:
According to the checklist, you should not ignore exceptions in try-catch blocks. While you are throwing a RuntimeException here, it's important to ensure that the message is informative enough. Consider including more details about the context of the error if possible, to make the exception as informative as it can be.